config/runtimes/kserve-mlserver.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: serving.kserve.io/v1alpha1
+kind: ClusterServingRuntime
+metadata:
+  name: kserve-mlserver
+spec:
+  supportedModelTypes:
+    - name: sklearn
+      version: "2"
```

Comment:
sklearn version should be "0"
xgboost version should be "1"
@adriangonz what are the other frameworks MLServer support?

Reply:
We could add MLflow and LightGBM to that list. Although I'm assuming that'd require changes in the code as well?

Reply:
@adriangonz the purpose of introducing servingruntime is that we no longer need code change for adding new frameworks.. we only need to create new serving runtime CR and install along with kserve